sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java

```diff
@@ -357,6 +360,14 @@ public void restrictPaginationByClass(Criteria criteria, String name, Consumer<S
         Restrictions.ilike("sampleClass.sampleCategory", name, MatchMode.START)));
   }
 
+  @Override
+  public void restrictPaginationByArrayed(Criteria criteria, boolean isArrayed, Consumer<String> errorHandler) {
```

Comment:
You don't actually use the parameter `isArrayed`.

Reply:
fixed